tests/helpers/test_condition.py

```diff
@@ -2235,7 +2236,7 @@ async def test_if_action_before_sunrise_no_offset(
     # sunrise: 2015-09-16 13:33:18 UTC,   sunset: 2015-09-17 01:53:45 UTC
     # now = sunrise + 1s -> 'before sunrise' not true
     now = datetime(2015, 9, 16, 13, 33, 19, tzinfo=dt_util.UTC)
-    with patch("homeassistant.util.dt.utcnow", return_value=now):
+    with freeze_time(now):
```

Comment:
Its generally a lot faster (order(s) of magnitude faster) to freeze time once, and than move it forward instead of doing a new context manager each time.  I'm not sure that would actually work here.

Reply:
I tried using the “FrozenDateTimeFactory” here but had no success.

Reply:
Its fine to move this forward as-is and improve it later.  Not patching dt.utcnow is already a great improvement as it makes the tests more reliable in the future